chutney/ui/src/app/core/services/scenario-execution.service.ts

```diff
@@ -87,11 +87,15 @@ export class ScenarioExecutionService {
             let es;
             try {
                 const token = localStorage.getItem('jwt') || sessionStorage.getItem('access_token');
-                es = new EventSourcePolyfill(url, {
-                    headers: {
-                        Authorization: `Bearer ${token}`
-                    }
-                });
+                var sseHeaders = {};
+                if (token && token.length > 0) {
+                    sseHeaders = {
+                        headers: {
+                            Authorization: `Bearer ${token}`
+                        }
+                    };
+                }
+                es = new EventSourcePolyfill(url, sseHeaders);
```

Comment:
If token is null, the server will not receive the Authorization header, and the connection will proceed without authentication. That's what we want ?